src/components/common/MenuAppBar.tsx

```diff
@@ -119,7 +119,12 @@ export default function MenuAppBar(props: MenuAppBarProps) {
           </Typography>
         </Toolbar>
       </AppBar>
-      <Drawer anchor="left" open={isMenuOpen} onClose={toggleMenu}>
+      <Drawer
+        anchor="left"
+        open={isMenuOpen}
+        onClose={toggleMenu}
+        disableScrollLock
```

Comment:
This was shifting a lot of buttons as it was adding right padding to our `body` container.